### PR TITLE
Allow updates to be disabled.

### LINKED ISSUE #565: Allow disabling OTA updates
As a privacy-conscious user I want to make sure that my gateway never updates by disabling OTA updates.

### PR DESCRIPTION
**Pull request by gucci-on-fleek**
Some platforms (Docker, package managers) expect to handle all updates. If the gateway updates itself, it could lead to inconsistencies. I've added a config flag to disable updates so that the native package manager (Docker, apt, pacman, yum, etc.) can handle them itself.

Fixes #565

### CODE REVIEW

File: static/js/views/settings.js [hunk 1]
```diff
@@ -1742,7 +1745,11 @@ const SettingsScreen = {
         statusText += fluent.getMessage('never');
       }
 
+      if (support.support) {
```
Comment (mrstegeman) [inline static/js/views/settings.js]: nit: indent line 1749

**Comment by gucci-on-fleek** [comment]
Alright, now the default is to disable updates, and updates are only enabled on Raspbian.

**Comment by mrstegeman** [comment]
Please merge https://github.com/gucci-on-fleek/gateway/pull/1

**Comment by gucci-on-fleek** [comment]
Merged

**Pull request merged**